module/Activity/src/Activity/Service/Signup.php

```diff
@@ -355,23 +411,36 @@ public function isAllowedToSubscribe()
      */
     public function isAllowedToExternalSubscribe()
     {
-        return $this->isAllowed('externalSignup', 'activitySignup');
+        return $this->isAllowed('externalSignup', 'signupList');
     }
 
     public function isAllowedToViewSubscriptions()
     {
-        return $this->isAllowed('view', 'activitySignup');
+        return $this->isAllowed('view', 'signupList');
     }
 
     public function isAllowedToInternalSubscribe()
```

Comment:
This method does exactly the same as isAllowedToSubscribe() on line 402.
Moreover, why do we need functions for performing this check since it is just one line of code?

Reply:
Last comment also holds for other isAllowedToFoo methods